WavesCS/Transactions/Transaction.cs

```diff
@@ -20,7 +21,27 @@ protected Transaction(byte[] senderPublicKey)
             Timestamp = DateTime.UtcNow;                        
             SenderPublicKey = senderPublicKey;            
             Proofs = new byte[8][];
-        }       
+        }
+
+        protected Transaction(DictionaryObject tx)
+        {
+            Timestamp = tx.GetDate("timestamp");
+
+            SenderPublicKey = tx.GetString("senderPublicKey").FromBase58();
+
```

Comment:
Ler's introduce Sender property in Transaction class as well. It's filled when transaction comes from the node and can be useful.